electron/src/js/menu/developer.ts

```diff
@@ -17,27 +17,27 @@
  *
  */
 
-const {MenuItem} = require('electron');
-const config = require('./../config');
-const environment = require('./../environment');
-const util = require('./../util');
-const windowManager = require('./../window-manager');
+import * as Electron from 'electron';
+import * as config from '../config';
+import * as environment from '../environment';
+import * as util from '../util';
+import * as windowManager from '../window-manager';
 
 const currentEnvironment = environment.getEnvironment();
 
 const getPrimaryWindow = () => windowManager.getPrimaryWindow();
 
-const reloadTemplate = {
+const reloadTemplate: Electron.MenuItemConstructorOptions = {
   click: () => getPrimaryWindow().reload(),
   label: 'Reload',
 };
 
-const devToolsTemplate = {
+const devToolsTemplate: Electron.MenuItemConstructorOptions = {
   label: 'Toggle DevTools',
   submenu: [
     {
       accelerator: 'Alt+CmdOrCtrl+I',
-      click: () => getPrimaryWindow().toggleDevTools(),
+      click: () => (getPrimaryWindow() as any).toggleDevTools(),
```

Comment:
This should not be any. Primary window is returning a `BrowserWindow`.

Reply:
True, but unfortunately the Electron types don't have `toggleDevTools()` included even though it exists in the JS code.